src/lib/github-api.js

```diff
@@ -1,21 +1,14 @@
-const Jira = require('./jira');
 const GitHub = require('github-api');
 const { GITHUB_TOKEN } = process.env;
 const gh = new GitHub({ token: GITHUB_TOKEN });
 const { Octokit } = require('@octokit/rest');
 const aws = require('./aws');
 const { log } = require('./utils');
 
-const https = require('https');
-const url = require('url');
-
-const { REPOS_URL = 'https://public.stoic.com/internal/meta/repositories-bot-github.json' } = process.env;
-
-const jira = Jira.initialize();
+const REPOS = ["core.stoic","fermat","kyu","praxis","principia","stoic-duckdb","stoic-io","team","demos.stoic","marcus","particula","lorem","pictura","grid","librarium","stoic-kubernetes","Utilities.stoic","Components.stoic","Datatypes.stoic","Transforms.stoic","AWS.stoic","Worker.stoic","Playground.stoic","Visuals.stoic","Transformations.stoic","Engine.stoic"];
```

Comment:
Remove `team`, order and prettify.
```suggestion
const REPOS = [
  'AWS.stoic',
  'Components.stoic',
  'core.stoic',
  'Datatypes.stoic',
  'demos.stoic',
  'Engine.stoic',
  'fermat',
  'grid',
  'kyu',
  'librarium',
  'lorem',
  'marcus',
  'particula',
  'pictura',
  'Playground.stoic',
  'praxis',
  'principia',
  'stoic-duckdb',
  'stoic-io',
  'stoic-kubernetes',
  'Transformations.stoic',
  'Transforms.stoic',
  'Utilities.stoic',
  'Visuals.stoic',
  'Worker.stoic',
];
```